### DISCUSSION TITLE
How to change Display Template for User collection

### DISCUSSION THREAD

**Discussion by 66Leo66** [directus/directus#17570] [Archive]
I'd like to change how users' names are displayed at login, M2M etc. For example, instead of using first+last name I might want to use nuckname or something.

As far as I can tell, `directus_users` collection's builtin fields are completely read-only.

I'd like to know if there is/will be a workaround e.g. editing database/use translate string.

I'd also suggest that certain customizations coule be allowed for system collections.

**Comment by iksent** [comment]
I need this too!

**Comment by JamesW1** [comment]
Thanks for the suggestion! We've moved feature requests over to our public roadmap at https://roadmap.directus.io/roadmap

I've logged this request for you, and if we see it raised multiple times, we'll consider adding it to the roadmap for prioritization.
